src/Class.ts

```diff
@@ -0,0 +1,60 @@
+type Session {
+    _id: string;
+    duration: number;
+    userId: string;
+}
+​
+class Countly {
+    public client;
+    public API_KEY = 'adf8739adf00ad9adsf9808d';
+    public CLIENT_ID = '234asdfa99dc'
```

Comment:
Can these members vars be set to private instead of public? Also consider the same for `getSessions` and `getUsers`